src/main/java/com/google/codeu/servlets/MapsServlet.java

```diff
@@ -0,0 +1,73 @@
+package com.google.codeu.servlets;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import java.io.IOException;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+/**
+ * Returns Starbucks Locations a JSON array, e.g. [{"lat": 38.4404675, "lng": -122.7144313}]
+ */
+
+@WebServlet("/starbucks-data")
+public class MapsServlet extends HttpServlet {
+  JsonArray starbucksLocationsArray;
+  
+  /**
+   *Method to parse through a comma serpearted string.
```

Comment:
Comments that can fit on one line should be put on one line. Comment should be descriptive of what the function is doing, parsing through a string is too vague. Consider the revision below:

`/** Method to parse through a comma separate string representing a Starbucks location. */`